src/mezz/prot-tls.r

```diff
@@ -2,7 +2,7 @@ REBOL [
 	title: "REBOL3 TLSv1.2 protocol scheme"
 	name: 'tls
 	type: 'module
-	author: rights: ["Richard 'Cyphre' Smolak" "Oldes" "HF"]
+	author: rights: ["Richard 'Cyphre' Smolak" "Oldes" "Brian Dickens (Hostilefork)"]
```

Comment:
@hostilefork @gchiu Sorry... will include your full name in future. Also as there is quite a lot words from you, I should note, that I started on [_almost_ full rewrite](https://github.com/Oldes/Rebol3/commit/30e89acd0bb4c102be0883c813196a4e8e71d779) of the TLS scheme much sooner than you released your Ren-C version. When you released it, I was examining it and used some of your ideas, so I added your name into authors. But still, anyone can compare your and my version and see, that these are very different. 

I don't understand rest of the message, especially who is mad and why. But that is ok. I should also say, that I rewrited TLS not just because I wanted some _community bounty_ for making it, but mainly because I wanted to learn, how the TLS work.

And when I'm in deserving credits, I should also thank to Eduard Suica, as I used his [tlse](https://github.com/eduardsui/tlse) project to learn quite a lot.

PS: I'm even not sure, if my version is fully functional, as I'm focused in other directions for a while, but it is always nice to have some contributors ;-) Even when it is just for correcting names.